crs_application/src/task_managers/part_registration_manager.cpp

```diff
@@ -276,14 +278,41 @@ common::ActionResult PartRegistrationManager::applyTransform()
     tf2::fromMsg(p, p_eig);
     return tf2::toMsg(t_eig * p_eig);
   };
-  for (auto& poses : cropped_raster_strips)
+  for (auto& poses : original_rasters)
   {
     for (std::size_t i = 0; i < poses.poses.size(); i++)
     {
       poses.poses[i] = apply_transform(poses.poses[i], part_transform_.transform);
     }
   }
-  result_.rasters = cropped_raster_strips;
+
+  geometry_msgs::msg::TransformStamped transform;
+  try
+  {
+    transform = tf_buffer_.lookupTransform(
+        original_rasters[0].header.frame_id, "base_link", tf2::TimePointZero, tf2::Duration(5));
+  }
+  catch (tf2::TransformException ex)
+  {
+    std::string error_msg =
```

Comment:
In the future you could use boost::format but it's ok as is.

Reply:
Noted